### PR TITLE
Add listings update / delete endpoints

### PR DESCRIPTION
**Pull request by andr-dev**

### CODE REVIEW

**Review by abhaykshar — APPROVED** [review]
thanks!

File: android/app/src/main/java/org/uwaterloo/subletr/api/apis/ListingsApi.kt [hunk 1]
```diff
@@ -125,6 +126,74 @@ class ListingsApi(basePath: kotlin.String = defaultBasePath, client: OkHttpClien
...
+     * @return kotlin.String
+     * @throws IllegalStateException If the request is not correctly configured
+     * @throws IOException Rethrows the OkHttp execute method exception
+     * @throws UnsupportedOperationException If the API returns an informational or redirection response
+     * @throws ClientException If the API returns a client error response
+     * @throws ServerException If the API returns a server error response
+     */
+    @Suppress("UNCHECKED_CAST")
+    @Throws(IllegalStateException::class, IOException::class, UnsupportedOperationException::class, ClientException::class, ServerException::class)
+    suspend fun listingsDelete() : kotlin.String = withContext(Dispatchers.IO) {
```
Comment (leeminkyu16) [inline android/app/src/main/java/org/uwaterloo/subletr/api/apis/ListingsApi.kt:141]: Just to confirm, this just deletes your listing, right?
If I can somewhat read Rust, I want to say yes....

**Review by leeminkyu16 — APPROVED** [review]
Let's Gamble, Try Merging

File: android/app/src/main/java/org/uwaterloo/subletr/api/apis/ListingsApi.kt (hunk 1 above)
Comment (andr-dev) [reply android/app/src/main/java/org/uwaterloo/subletr/api/apis/ListingsApi.kt:141]: Yep!

**Pull request merged**